src/core/lib/iomgr/closure.h

```diff
@@ -42,7 +42,9 @@ typedef struct grpc_closure_list {
  *
  * \param arg Arbitrary input.
  * \param error GRPC_ERROR_NONE if no error occurred, otherwise some grpc_error
- *              describing what went wrong */
+ *              describing what went wrong.
+ *              Error contract: it is not the cb's job to unref this error;
```

Comment:
FWIW, note that this is already documented here:

https://github.com/grpc/grpc/blob/master/doc/core/grpc-error.md